src/coord/src/coord/dataflow_builder.rs

```diff
@@ -164,6 +167,29 @@ impl<'a> DataflowBuilder<'a, mz_repr::Timestamp> {
 
                         let source_description = self.catalog.source_description_for(*id).unwrap();
 
+                        let source_description = if let SourceConnector::External {
```

Comment:
What's going on here? Toss a comment in, so that future maintainers can grok what needs doing.

Reply:
I'm rolling this change back because it's not needed right now. I had it because I introduced a `SourceConnector::Persist` variant but we should do that later, if at all.